apps/competitions-api/src/routes/inscriptions/updateInscription.ts

```diff
@@ -0,0 +1,61 @@
+import { Router } from 'express';
+import { parseRequest, CustomRequest, checkAdminRole, checkRole, Key, catchError, logRequestMiddleware } from '@competition-manager/backend-utils';
+import { UpdateInscription$, BaseAdmin$, Access, Role, competitionInclude } from '@competition-manager/schemas';
+import { z } from 'zod';
+import { prisma } from '@competition-manager/prisma';
+import { logger } from '../../logger';
+import { isAuthorized } from '@competition-manager/utils';
+
+export const router = Router();
+
+const Params$ = z.object({
+    competitionEid: z.string(),
+    inscriptionEid: z.string()
+})
+
+router.put(
+    '/:competitionEid/inscriptions/:inscriptionEid',
+    logRequestMiddleware(logger),
+    parseRequest(Key.Params, Params$),
+    parseRequest(Key.Body, UpdateInscription$),
+    checkRole(Role.ADMIN),
+    async (req: CustomRequest, res) => {
+        try {
+            const { competitionEid, inscriptionEid } = Params$.parse(req.params);
+            const inscriptionData = UpdateInscription$.parse(req.body);
+            const competition = await prisma.competition.findUnique({
+                where: { eid: competitionEid },
+                include: competitionInclude
+            });
+            if (!competition) {
+                res.status(404).send('competitionNotFound');
```

Comment:
pas bonne key à mon avis 
```suggestion
                res.status(404).send('errors.competitionNotFound');
```